src/views/LocationInput.jsx

```diff
@@ -1,31 +1,11 @@
 import { AppHeader } from './Header';
-import { Container, Notification } from 'bloomer';
 import { Foot } from './Footer';
 import { LocationInputForm } from './LocationInputForm';
-import { connect } from '@cerebral/react';
-import { state } from 'cerebral';
 import React from 'react';
-export const LocationInput = connect(
-  {
-    failure: state.submitLocationFailure,
-    submitMsg: state.submitLocationMsg,
-    success: state.submitLocationSuccess,
-  },
-  ({ failure, submitMsg, success }) => {
-    let color = success ? 'success' : 'danger';
-    return (
-      <React.Fragment>
-        <AppHeader />
-        <LocationInputForm />
-        {(success || failure) && (
-          <Container>
-            <Notification isColor={color}>
-              <h1 className="location-name">{submitMsg}</h1>
-            </Notification>
-          </Container>
-        )}
-        <Foot />
-      </React.Fragment>
-    );
-  },
+export const LocationInput = () => (
+  <React.Fragment>
```

Comment:
_nit_: save bytes, instead of `<React.Fragment>` use `<>` and `</>` if you're so inclined :)

Reply:
I was wondering about that - I saw `<>` used in the USTC code, but not in this codebase. Thank you!